exporter/osg/osgbake.py

```diff
@@ -47,6 +48,43 @@ def collectValues():
             for block in morph_info:
                 morph_info[block].append(block.value)
 
+    def evaluateActiveShapeKeys(time, times):
+        # Get effective shapes according to current time
+        # and evaluate the factor for each
+        value = 0.0
```

Comment:
I'd rather put the declaration in the `if` branch as all 3 cases will have a specific behavior